### PR TITLE
Temporarily reduce e2e test matrix to stop flaky web engine builds

### PR DESCRIPTION
**Pull request by yjbanov**
## Description

Temporarily reduce e2e test matrix to stop flaky web engine builds.

This should be reverted when we have shards available for LUCI web builds.

## Related Issues

Fixes https://github.com/flutter/flutter/issues/71576

### CODE REVIEW

**Review by nturgut — COMMENTED** [review]
Why didn't we added some tests to the blocked list as we discussed on Monday?

I think that was our decision after the conversation.

**Comment by yjbanov** [comment]
> Why didn't we added some tests to the blocked list as we discussed on Monday?
> 
> I think that was our decision after the conversation.

Done.

File: lib/web_ui/dev/integration_tests_manager.dart [hunk 1]
```diff
@@ -632,9 +632,26 @@ const Map<String, List<String>> blockedTestsListsMapForModes =
     'treeshaking_integration.dart',
     'text_editing_integration.dart',
     'url_strategy_integration.dart',
+
+    // The following tests are blocked to reduce the load on the build bot.
```
Comment (zanderso) [inline lib/web_ui/dev/integration_tests_manager.dart:636]: Is there an issue filed to track re-enabling these that you can note here?

**Review by cbracken — APPROVED** [review]
lgtm with same nit as @zanderso. Add a TODO to re-enable with an issue link.

File: lib/web_ui/dev/integration_tests_manager.dart (hunk 1 above)
Comment (yjbanov) [reply lib/web_ui/dev/integration_tests_manager.dart:636]: Filed https://github.com/flutter/flutter/issues/71583

**Comment by yjbanov** [comment]
@cbracken 

> lgtm with same nit as @zanderso. Add a TODO to re-enable with an issue link.

Done.

**Review by zanderso — APPROVED** [review]

**Review by nturgut — APPROVED** [review]
LGTM.

Text editing broke before just for release version btw. So I'll probably remove it back as soon as possible. (I remember there was also a case like that for image loading but I might be mistaken)

**Comment by yjbanov** [comment]
Merging on red as this PR aims to fix the current redness.

**Pull request merged**